styleguide.config.js

```diff
@@ -28,7 +28,7 @@ function generateSection({ componentNames, name, content }) {
 }
 
 module.exports = {
-  title: "Reaction Design System",
```

Comment:
Check the text in the tab:

<img width="995" alt="Screen Shot 2019-07-31 at 1 33 43 PM" src="https://user-images.githubusercontent.com/3673236/62246127-d666b680-b397-11e9-939a-d7442a78254d.png">